app/src/org/commcare/dalvik/dialogs/PaneledChoiceDialog.java

```diff
@@ -0,0 +1,111 @@
+package org.commcare.dalvik.dialogs;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.graphics.drawable.Drawable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import org.commcare.dalvik.R;
+
+/**
+ * Created by amstone326 on 10/27/15.
+ */
+public class PaneledChoiceDialog {
+
+    private View view;
+    private Context context;
+    private AlertDialog dialog;
+    private boolean usingThreePanelView;
+
+    public static boolean HORIZONTAL_THREE_PANEL = true;
+
+    public PaneledChoiceDialog(Context context, String title) {
+        this(context, title, false);
+    }
+
+    public PaneledChoiceDialog(Context context, String title, boolean useSpecialThreePanelView) {
```

Comment:
If you aren't into making a static factory like `buildThreeChoiceDialog`, it might be even better to just create a subclass that appropriately overrides the constructor and `setChoiceItems` method such that you don't need the `usingThreePanelView` or `HORIZONTAL_THREE_PANEL` fields